specification/DigitalOcean-public.v2.yaml

```
@@ -1185,5 +1185,105 @@ components:
     bearer_auth:
       type: http
       scheme: bearer
+      description: |
+        ## OAuth Authentication
+
+        In order to interact with the DigitalOcean API, you or your application must
+        authenticate.
+
+        The DigitalOcean API handles this through OAuth, an open standard for
+        authorization. OAuth allows you to delegate access to your account in full
+        or in read-only mode.
+
+        You can generate an OAuth token by visiting the [Apps & API](https://cloud.digitalocean.com/settings/applications)
+        section of the DigitalOcean control panel for your account.
+
+        An OAuth token functions as a complete authentication request. In effect, it
+        acts as a substitute for a username and password pair.
+
+        Because of this, it is absolutely **essential** that you keep your OAuth
+        tokens secure. In fact, upon generation, the web interface will only display
+        each token a single time in order to prevent the token from being compromised.
+
+        ### How to Authenticate with OAuth
+
+        In order to make an authenticated request, include a bearer-type
+        `Authorization` header containing your OAuth token. All requests must be
+        made over HTTPS.
+
+        #### Authenticate with a Bearer Authorization Header
+
+        ```
+        curl -X $HTTP_METHOD -H "Authorization: Bearer $TOKEN" "https://api.digitalocean.com/v2/$OBJECT"
```

```suggestion
        curl -X $HTTP_METHOD -H "Authorization: Bearer $DIGITALOCEAN_TOKEN" "https://api.digitalocean.com/v2/$OBJECT"
```

Let's use `DIGITALOCEAN_TOKEN` to be consistent with the examples.